pkg/aggregator/aggregator.go

```diff
@@ -732,7 +737,23 @@ func (agg *BufferedAggregator) run() {
 			agg.hostname = h
 			changeAllSendersDefaultHostname(h)
 			agg.hostnameUpdateDone <- struct{}{}
+		case orchestratorMetadata := <-agg.orchestratorMetadataIn:
+			aggregatorOrchestratorMetadata.Add(1)
+			// each resource has its own payload so we cannot aggregate
```

Comment:
with aggregate you mean aggregating the messages until we flush them like here? https://github.com/DataDog/datadog-agent/blob/93059ab4f6bc5a2d6349b52b21ea042ed8e485ea/pkg/aggregator/aggregator.go#L636-L644

Reply:
yes 👍